index.js

```diff
@@ -0,0 +1,97 @@
+const HOME_MENU_BUTTONS_ARRAY = [
+    {
+        id: "internet-contents",
+        className: "home-options",
```

Comment:
if className is always the same, then there is no need to specify it with each element

Reply:
Solved